cryptofeed/exchange/bitmex.py

```diff
@@ -29,31 +29,44 @@
 class Bitmex(Feed):
     id = BITMEX
     api = 'https://www.bitmex.com/api/v1/'
-    symbol_endpoint = "https://www.bitmex.com/api/v1/instrument/active"
+    symbol_endpoint = "https://www.bitmex.com/api/v1/instrument/activeAndIndices"
 
     @classmethod
     def _parse_symbol_data(cls, data: dict, symbol_separator: str) -> Tuple[Dict, Dict]:
         ret = {}
         info = defaultdict(dict)
 
         for entry in data:
-            components = []
-            components.append(entry['rootSymbol'])
-            components.append(entry['quoteCurrency'])
+            exchangeSymbol = entry['symbol']
+            if exchangeSymbol[0] == '.':
+                # Need to replace the dot on the index symbol
+                exchangeSymbol = f'{INDEX_PREFIX}{exchangeSymbol[1:]}'
+                normalized = exchangeSymbol
+                normalized = normalized.replace("XBT", "BTC")
+                info['index'][normalized] = True
```

Comment:
I dislike the fact that we have to do this kind of translation between the exchange symbol (which am I right is the one we use in NATS) and the symbol used in WE requests. In scenarios like these, using the CF standard symbol for NATS would make things a lot easier as the exchange symbol could only be used in the exchange and wouldn't have to care about NATS formatting.